src/hotspot/share/jvmci/vmStructs_jvmci.cpp

```diff
@@ -495,6 +495,12 @@
 #define VM_INT_CONSTANTS(declare_constant, declare_constant_with_value, declare_preprocessor_constant) \
   declare_preprocessor_constant("ASSERT", DEBUG_ONLY(1) NOT_DEBUG(0))     \
                                                                           \
+  declare_preprocessor_constant("INCLUDE_SERIALGC",     INCLUDE_SERIALGC)     \
```

Comment:
Probably best to make the formatting consistent with how it's done for the `JVM_ACC_*` constants (i.e., no alignment of values).